userguide/advanceduse/reverse-tethering.rst

```diff
@@ -0,0 +1,32 @@
+Reverse tethering
+=================
+
+Some users may don't have an available wifi connection for their phone nor a data subscription from their mobile provider.
+This short tutorial will help you to connect your Ubuntu Touch to your computer to access internet.
+
+Steps:
+------
+
+1 - on phone: ``android-gadget-service enable rndis``
```

Comment:
This numbering style does not work in the build output. Please use the following:

1. 

2.